### PR TITLE
python312Packages.crc32c: 2.4 -> 2.7.1

### PR DESCRIPTION
**Pull request by bcdarwin**
Routine update [changelog](https://github.com/ICRAR/crc32c/blob/master/CHANGELOG.md#271)

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by bcdarwin** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 376816`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 8 packages marked as broken and skipped:</summary>
  <ul>
    <li>piper-train</li>
    <li>piper-train.dist</li>
    <li>python312Packages.elegy</li>
    <li>python312Packages.elegy.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
    <li>python313Packages.parsedmarc</li>
    <li>python313Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python313Packages.finetuning-scheduler</li>
    <li>python313Packages.finetuning-scheduler.dist</li>
    <li>python313Packages.kafka-python-ng</li>
    <li>python313Packages.kafka-python-ng.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 26 packages built:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python312Packages.crc32c</li>
    <li>python312Packages.crc32c.dist</li>
    <li>python312Packages.fastmri</li>
    <li>python312Packages.fastmri.dist</li>
    <li>python312Packages.finetuning-scheduler</li>
    <li>python312Packages.finetuning-scheduler.dist</li>
    <li>python312Packages.kafka-python-ng</li>
    <li>python312Packages.kafka-python-ng.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-lightning</li>
    <li>python312Packages.pytorch-lightning.dist</li>
    <li>python312Packages.tensorboardx</li>
    <li>python312Packages.tensorboardx.dist</li>
    <li>python313Packages.crc32c</li>
    <li>python313Packages.crc32c.dist</li>
    <li>python313Packages.fastmri</li>
    <li>python313Packages.fastmri.dist</li>
    <li>python313Packages.pytorch-lightning</li>
    <li>python313Packages.pytorch-lightning.dist</li>
    <li>python313Packages.tensorboardx</li>
    <li>python313Packages.tensorboardx.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

**Comment by bcdarwin** [comment]
The four breakages python313Packages.{finetuning-scheduler,kafka-python-ng}{,.dist} already don't build upstream due to python 3.13 issues.
